M2/Macaulay2/packages/VirtualResolutions/doc.m2

```diff
@@ -1,7 +1,4 @@
-undocumented {
-    (multigradedRegularity, Thing, Ideal),
-    (multigradedRegularity, Thing, Thing, Module)
-    }
+undocumented { "LowerLimit", "UpperLimit" }
```

Comment:
document these

If they are options to just one method, just point the documentation to the same node.  If not, just say "This is a symbol used as an optional argument name."  Or something like that.